api/middleware/auth0Middleware.js

```diff
@@ -0,0 +1,64 @@
+const { expressjwt: jwt } = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
+const createError = require('http-errors');
+const config = require('../../config/auth0');
+// const { findOrCreateProfile } = require('../profile/profileModel');
```

Comment:
you can feel at liberty to remove this commented out code and the lines below it as well.